bundle/mpas-jedi-data/testinput_tier_1/obs

```diff
@@ -0,0 +1 @@
+../../../fix/jcsda/mpas-jedi-data/testinput_tier_1/obs
```

Comment:
I'm curious why this is linked in through `bundle/` while the other test dats (`fv3-jedi-data` and `ioda-data`) are linked in through `fix/jcsda`.

Reply:
I don't quite get this question. 
I assume you are curious why there a new `obs` link under `bundle/mpas-jedi-data/testinput_tier_1/obs`? 

If so, it is because a legacy issue how the mpas-jedi ctests was set up in RDASApp. Although the mpas-jedi ctests will not write anything to `bundle/mpas-jedi-data`, it require this directory to be writable. So `mpas-jedi-data` directory has to treated specially.

Reply:
I understand now, thanks.